tools/regenerateAll.bat

```diff
@@ -0,0 +1,72 @@
+@echo off
+set repoUser=olydis
+set repoBranch=batches
+set version=latest
+call ..\src\SDKs\StreamAnalytics\Management.StreamAnalytics\generate.cmd                              %version% %repoUser% %repoBranch% noInstall
+call ..\src\SDKs\Storage\Management.Storage\generate.cmd                                              %version% %repoUser% %repoBranch% noInstall
+:: ...
+call ..\src\SDKs\SqlManagement\Management.Sql\generate.cmd                                            %version% %repoUser% %repoBranch% noInstall
+call ..\src\SDKs\ServiceFabric\Management.ServiceFabric\generate.cmd                                  %version% %repoUser% %repoBranch% noInstall
+:: ...
+call ..\src\SDKs\ServerManagement\Management.ServerManagement\generate.cmd                            %version% %repoUser% %repoBranch% noInstall
+:: ...
+call ..\src\SDKs\Resource\Management.ResourceManager\generate.cmd                                     %version% %repoUser% %repoBranch% noInstall
+call ..\src\SDKs\RedisCache\Management.Redis\generate.cmd                                             %version% %repoUser% %repoBranch% noInstall
+:: ?
+call ..\src\SDKs\RecoveryServices\Management.RecoveryServices\generate.cmd                            %version% %repoUser% %repoBranch% noInstall
+:: renames, version, ?
+call ..\src\SDKs\RecoveryServices.SiteRecovery\Management.RecoveryServices.SiteRecovery\generate.cmd  %version% %repoUser% %repoBranch% noInstall
+call ..\src\SDKs\RecoveryServices.Backup\Management.RecoveryServices.Backup\generate.cmd              %version% %repoUser% %repoBranch% noInstall
+:: ?
+call ..\src\SDKs\PowerBIEmbedded\Management.PowerBIEmbedded\generate.cmd                              %version% %repoUser% %repoBranch% noInstall
+:: stuff missing

```

Comment:
@olydis this seems like a test for making sure generation works?
If that is true? at what point SDK branch should run this test?